recipes/miniFASTA/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "miniFasta" %}
+{% set version = "2.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/not-a-feature/miniFASTA/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fe83656e252306ec9683efc90ece2f20fcf78806d285b9056a02096ecb120e12
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - miniFasta
```

```suggestion
test:
  imports:
    - miniFasta
  commands:
    - pip check
  requires:
    - pip
```
This just double checks the reqs.